src/tactic/default.lean

```diff
@@ -30,5 +30,4 @@ import
   tactic.wlog
   tactic.tfae
   tactic.apply_fun
-  tactic.localized
```

Comment:
Sorry if you said somewhere, why is this going?

Reply:
it's already imported in `tactic.basic`.